diag_manager/fms_diag_file_object.F90

```diff
@@ -1203,6 +1211,16 @@ subroutine increase_unlimited_dimension(this)
   this%FMS_diag_file%unlimited_dimension = this%FMS_diag_file%unlimited_dimension + 1
 end subroutine increase_unlimited_dimension
 
+!> \brief Get the unlimited dimension that is in the file
+!! \return The unlimited dimension
```

Comment:
This is an integer.  Is this an ID?  Is this the current value of the dimension?